wasm-glow-ballers-lib/src/lib.rs

```diff
@@ -5,13 +5,109 @@ use std::collections::HashMap;
 use error::Error;
 use walrus::Module;
 
-pub fn mapping(wasm: &[u8]) -> Result<HashMap<String, usize>, Error> {
+pub enum MappingItemType {
```

Comment:
Deriving the following for pub types makes the library more usable for others:

`#[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]`